### PR TITLE
Bump version to v0.10.0

### PR DESCRIPTION
**Pull request by robyoung**
Depends on #50

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,5 +30,6 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Fix rustdoc warnings
 - Upgrade nrf51-hal to 0.12.1
 
-[Unreleased]: https://github.com/therealprof/microbit/compare/v0.9.0...HEAD
-[0.9.0]: https://github.com/therealprof/microbit/compare/v0.8.0...v0.9.0
+[Unreleased]: https://github.com/nrf-rs/microbit/compare/v0.10.0...HEAD
+[0.9.0]: https://github.com/nrf-rs/microbit/compare/v0.8.0...v0.9.0
```
Comment (therealprof) [inline CHANGELOG.md:34]: Can you swith `0.9.0` and `0.10.0` so it the order is latest to oldest?

**Comment by therealprof** [comment]
Any instructions on how to release this?

**Comment by robyoung** [comment]
> Any instructions on how to release this?

I'm not sure I'm afraid. I would guess you just `cargo publish` in each workspace member but I've never been involved in publishing a workspace project like this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -26,5 +30,6 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Fix rustdoc warnings
 - Upgrade nrf51-hal to 0.12.1
 
-[Unreleased]: https://github.com/nrf-rs/microbit/compare/v0.9.0...HEAD
+[Unreleased]: https://github.com/nrf-rs/microbit/compare/v0.10.0...HEAD
+[0.10.0]: https://github.com/nrf-rs/microbit/releases/tag/v0.10.0...v0.9.0
```
Comment (therealprof) [inline CHANGELOG.md:34]:
```suggestion
[0.10.0]: https://github.com/nrf-rs/microbit/releases/tag/v0.9.0...v0.10.0
```
Close, but no cigar. 😉

Comment (robyoung) [reply CHANGELOG.md]: aww man! I'm doing the fixes in `xtask bump`, regenerating and then rebasing so typos are getting through. Hopefully this time. 🤞🏻

**Review by therealprof — APPROVED** [review]
LGTM, let's give this a try.

bors r+

**Comment by therealprof** [comment]
bors ping

**Comment by therealprof** [comment]
@robyoung Okay, there're a couple of pitfalls publishing this.

1. microbit-common needs to be published first
2. `microbit-common`, `microbit`, `microbit-v2` and the examples cannot be compiled (and thus not published with verifcation) using either `v1` or `v2` features on my Mac:
```
LLVM ERROR: Global variable 'Reset' has an invalid section specifier '.Reset': mach-o section specifier requires a segment and section separated by a comma.
error: could not compile `cortex-m-rt`

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
LLVM ERROR: Global variable '__INTERRUPTS' has an invalid section specifier '.vector_table.interrupts': mach-o section specifier requires a segment whose length is between 1 and 16 characters.
error: failed to verify package tarball
```

**Comment by robyoung** [comment]
Is it worth wrapping the steps up in another xtask so that it's easier in future? It already has code to ensure all the required targets are installed.

**Pull request merged**